packages/ocaml-migrate-parsetree-android.1.0.11/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "1.2"
+maintainer: "frederic.bour@lakaban.net"
+authors: [
+  "Frédéric Bour <frederic.bour@lakaban.net>"
+  "Jérémie Dimino <jeremie@dimino.org>"
+]
+license: "LGPL-2.1"
+homepage: "https://github.com/ocaml-ppx/ocaml-migrate-parsetree"
+bug-reports: "https://github.com/ocaml-ppx/ocaml-migrate-parsetree/issues"
+dev-repo: "git://github.com/ocaml-ppx/ocaml-migrate-parsetree.git"
+tags: [ "syntax" "org:ocamllabs" ]
+build: [
+  ["jbuilder" "build" "-p" "ocaml-migrate-parsetree" "-j" jobs]
+]
+install: [
```

Comment:
I don't think you need `install` or `remove` clauses with jbuilder. @toots?

Reply:
not if the package is able to produce the `.install` files.

Reply:
Removed.